internal/jimm/bootstrap/bootstrap_test.go

```diff
@@ -235,22 +243,22 @@ func (s *bootstrapManagerSuite) TestBootstrapJob(c *qt.C) {
 		binary.Done()
 	})
 
-	ctrl, store, jujuManager, binaryStore, executor, clientStore, user := setupMocks(c)
+	ctrl, mocks, user := setupTest(c)
 	defer ctrl.Finish()
 
-	manager, err := bootstrap.NewBootstrapManager(store, s.jobTracker, jujuManager, binaryStore, loginTokenRefreshURLParam)
+	manager, err := bootstrap.NewBootstrapManager(mocks.store, s.jobTracker, mocks.jujuManager, mocks.binaryStore, loginTokenRefreshURLParam)
 	c.Assert(err, qt.IsNil)
 
 	// Mocked in order of execution:
 	cleanupCalled := false // To be asserted after job run - ensures cleanup was run.
-	store.EXPECT().LockBootstrap(gomock.Any(), gomock.Any()).Return(nil).Times(1)
```

Comment:
In Juju the standard is to drop usage of `Times(1)` because the default is to expect the call only once unless otherwise specified. So I'd like to stick with that.